### PR TITLE
2 harvesting features

### PR DESCRIPTION
**Pull request by mariano54**
- Warning level showing filename if quality lookup takes longer than 5 seconds
- Allow configuration of plot loading interval (default 2 minutes)

### CODE REVIEW

File: chia/harvester/harvester_api.py [hunk 1]
```diff
@@ -193,8 +195,17 @@ async def lookup_challenge(filename: Path, plot_info: PlotInfo) -> List[harveste
...
+        for filename_sublist_awaitable in asyncio.as_completed(awaitables):
+            filename, sublist = await filename_sublist_awaitable
+            time_taken = time.time() - start
+            if time_taken > 5:
+                self.harvester.log.warning(
+                    f"Looking up qualities on {filename} took: {time.time() - start}. This should be below 5 seconds "
+                    f"to minimize risk of losing rewards."
+                )
+            else:
+                self.harvester.log.debug(f"Looking up qualities on {filename} took: {time.time() - start}")
```
Comment (wjblanke) [inline chia/harvester/harvester_api.py:207]: This could potentially be humongous

File: chia/harvester/harvester_api.py [hunk 2]
```diff
@@ -193,8 +195,19 @@ async def lookup_challenge(filename: Path, plot_info: PlotInfo) -> List[harveste
...
         # Concurrently executes all lookups on disk, to take advantage of multiple disk parallelism
         total_proofs_found = 0
-        for sublist_awaitable in asyncio.as_completed(awaitables):
-            for response in await sublist_awaitable:
+        for filename_sublist_awaitable in asyncio.as_completed(awaitables):
+            filename, sublist = await filename_sublist_awaitable
+            time_taken = time.time() - start
+            if time_taken > 5:
+                self.harvester.log.warning(
+                    f"Looking up qualities on {filename} took: {time.time() - start}. This should be below 5 seconds "
```
Comment (wjblanke) [inline chia/harvester/harvester_api.py:203]: This is time since the start of the entire load operation, not the individual file lookup, correct? So u could see multiple of these for each file after passing 5 seconds. So maybe rephrase this and use time_taken?

Comment (wjblanke) [reply chia/harvester/harvester_api.py:203]: Also NAS guys will see this all the time. A lot run at 13 seconds.

Comment (mariano54) [reply chia/harvester/harvester_api.py:203]: I don't think so, because all of the lookups start at the same time. Each challenge is sent to a different thread

**Review by wjblanke — APPROVED** [review]
aok
